tests/datasource/fluent/integration/test_integration_datasource.py

```diff
@@ -515,35 +517,84 @@ def test_pandas_data_adding_dataframe_in_cloud_context(
     cloud_api_fake: RequestsMock,
     empty_cloud_context_fluent: CloudDataContext,
 ):
-    context = empty_cloud_context_fluent
-
     df = pd.DataFrame({"column_name": [1, 2, 3, 4, 5]})
 
+    context = empty_cloud_context_fluent
+
     dataframe_asset = context.sources.add_or_update_pandas(
         name="fluent_pandas_datasource"
-    ).add_dataframe_asset(name="my_df_asset", dataframe=df)
-    dataframe_asset.build_batch_request()
+    ).add_dataframe_asset(name="my_df_asset")
+    _ = dataframe_asset.build_batch_request(dataframe=df)
+    assert dataframe_asset.dataframe.equals(df)
+
+
+@pytest.mark.integration
+def test_pandas_data_adding_dataframe_in_file_reloaded_context(
+    empty_file_context: FileDataContext,
+):
+    df = pd.DataFrame({"column_name": [1, 2, 3, 4, 5]})
+
+    context = empty_file_context
+
+    datasource = context.sources.add_or_update_pandas(name="fluent_pandas_datasource")
+    dataframe_asset = datasource.add_dataframe_asset(name="my_df_asset")
+    context._save_project_config()
```

Comment:
`non-blocking`
This should automatically be called now after the asset is added.

Reply:
@Kilo59 I deleted it -- thanks.  Probably left over from trying to get the test to work, and thought that it was needed during one of the attempts, and then left it in there once things worked, without checking to see if it will work without that "save".  Thanks.